docs/dapp/opl/celer/ping-example.md

```diff
@@ -123,11 +123,11 @@ For compatibility with Sapphire, compile the contract using compiler version
 
 You can also use Celer's framework contracts and interfaces by importing them
 
-```solidity
-import "sgn-v2-contracts/contracts/message/framework/MessageBusAddress.sol";
-import "sgn-v2-contracts/contracts/message/framework/MessageReceiverApp.sol";
-import "sgn-v2-contracts/contracts/message/interfaces/IMessageBus.sol";
-```
+    ```solidity
+    import "sgn-v2-contracts/contracts/message/framework/MessageBusAddress.sol";
+    import "sgn-v2-contracts/contracts/message/framework/MessageReceiverApp.sol";
+    import "sgn-v2-contracts/contracts/message/interfaces/IMessageBus.sol";
+    ```
```

Comment:
Did you mean this identation?

```suggestion
```solidity
    import "sgn-v2-contracts/contracts/message/framework/MessageBusAddress.sol";
    import "sgn-v2-contracts/contracts/message/framework/MessageReceiverApp.sol";
    import "sgn-v2-contracts/contracts/message/interfaces/IMessageBus.sol";
```
```

Reply:
the md linter expects it indented not fenced 🤷‍♂️